sdk/nodejs/VERSION

```diff
@@ -1,3 +0,0 @@
-Generated by pulumi-tfgen-random from:
-Repo: github.com/terraform-providers/terraform-provider-random
-Commit: a3da7a23d7f9965c16311d203aee385c4c9e770c
```

Comment:
Should this file have been deleted?

Reply:
Great question... This file was last written 2 years ago. I hope this is a legacy thing that we don't use anymore...